src/Agent.Listener/_project.json

```diff
@@ -34,11 +34,11 @@
         "Microsoft.NETCore.App": "1.0.0-rc3-004306",
         "Microsoft.VisualStudio.Services.Agent": { "target": "project", "version": "1.0.0-*" },
         "Microsoft.Win32.Registry": "4.0.0-rc3-24125-01",
-        "vss-api-netcore": "0.5.10-private",
+        "vss-api-netcore": "0.5.18-private",
```

Comment:
taken care This change is not part of this PR anymore. Version 19 is already in.